cmake/emu/nsim.cmake

```diff
@@ -1,26 +1,62 @@
 # SPDX-License-Identifier: Apache-2.0
-if(${CONFIG_SOC_NSIM_HS_SMP})
+if("${BOARD_DEBUG_RUNNER}" STREQUAL "mdb-nsim" OR "${BOARD_FLASH_RUNNER}" STREQUAL "mdb-nsim")
 # mdb is required to run nsim multicore targets
 find_program(
   MDB
   mdb
   )
+set(MDB_BASIC_OPTIONS -nooptions -nogoifmain -toggle=include_local_symbols=1)
 
 if(${CONFIG_SOC_NSIM_HS_SMP})
  set(MDB_ARGS mdb_hs_smp.args)
+elseif(${CONFIG_SOC_NSIM_EM})
+ set(MDB_ARGS mdb_em.args)
+elseif(${CONFIG_SOC_NSIM_EM7D_V22})
+ set(MDB_ARGS mdb_em7d_v22.args)
+elseif(${CONFIG_SOC_NSIM_SEM})
+ set(MDB_ARGS mdb_sem.args)
+elseif(${CONFIG_SOC_NSIM_HS})
+ set(MDB_ARGS mdb_hs.args)
+elseif(${CONFIG_SOC_NSIM_HS6X})
+ set(MDB_ARGS mdb_hs6x.args)
+endif()
```

Comment:
Should we move it to `boards/arc/nsim/board.cmake`?

We do similar thing for runners arguments in it:
```
if(${CONFIG_SOC_NSIM_EM})
board_runner_args(arc-nsim "--props=nsim_em.props")
board_runner_args(mdb-nsim "--nsim_args=mdb_em.args")
elseif(${CONFIG_SOC_NSIM_EM7D_V22})
board_runner_args(arc-nsim "--props=nsim_em7d_v22.props")
board_runner_args(mdb-nsim "--nsim_args=mdb_em7d_v22.args")
elseif(${CONFIG_SOC_NSIM_SEM})
board_runner_args(arc-nsim "--props=nsim_sem.props")
board_runner_args(mdb-nsim "--nsim_args=mdb_sem.args")
elseif(${CONFIG_SOC_NSIM_HS})
board_runner_args(arc-nsim "--props=nsim_hs.props")
 [**snip**]
```